.github/workflows/publish.yml

```diff
@@ -15,13 +15,14 @@ jobs:
     strategy:
       matrix:
         env:
-        - NAME: bionic-release
-          OS_RELEASE: 18.04
         - NAME: focal-release
           OS_RELEASE: 20.04
     steps:
 
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
+      with:
+          submodules: recursive
+          fetch-depth: 0
```

Comment:
This can be done upstream

Reply:
It's not needed when building a single container image for all the tools.